fusedwind/turbine/test/data/DTU10MW.dp3d

```diff
@@ -1,4 +1,5 @@
 # web00  web01  web02  web03
+# top  mid  mid  mid
```

Comment:
That's also fine. I think sub versions are not necessary. How should we deal with the tests, should we test both file versions?
So far, we don't have a pure reader writer test. I could create one in `test_structure.py`.